src/demos/AirshipDropdown.tsx

```diff
@@ -83,33 +82,33 @@ export class AirshipDropdown extends React.Component<Props> {
     } = this.props
 
     return (
-      <LayoutContext>
+      <SafeAreaConsumer>
         {metrics => {
-          const { safeAreaInsets } = metrics
-
-          const screenStyle = {
-            bottom: safeAreaInsets.bottom,
-            left: safeAreaInsets.left,
-            right: safeAreaInsets.right,
-            top: 0
+          if(metrics) {
```

Comment:
I'd prefer an "early out" coding style, like `if (metrics == null) return null`, which prevents the rest of the function from indenting another level.